jetty-xml/src/test/java/org/eclipse/jetty/xml/XmlConfigurationTest.java

```diff
@@ -981,4 +981,17 @@ public void testJettyStandardIdsAndProperties_JettyHomeUri_JettyBaseUri() throws
             assertThat(propName, tc.getTestString(), startsWith("file:"));
         }
     }
+
+    @Test
+    public void testDeprecated() throws Exception
+    {
+        XmlConfiguration xmlConfiguration = new XmlConfiguration("" +
+            "<Configure class=\"org.eclipse.jetty.xml.AnnotatedTestConfiguration\">" +
+            "  <Set name=\"deprecated\">foo</Set>" +
+            "  <Call name=\"setDeprecated\"><Arg><Get name=\"deprecated\" /></Arg></Call>" +
+            "</Configure>");
+
+        xmlConfiguration.configure();
+        // Cannot test that the warnings are logged.
+    }
```

Comment:
Are you sure about that? Can you configure the log output into a file and then check the lines are in the file? If we can't automatically execute the test meaningfully, is it actually worth having a test?

Reply:
@janbartel the test is worth in that it should not fail or throw just because we're calling/handling deprecated methods.
@joakime any idea how I would test the output?

Reply:
@janbartel addresses your review, now testing the log output too.

Reply:
There's also StdErrCapture in jetty-util.

Example of how it is used ...

https://github.com/eclipse/jetty.project/blob/jetty-9.4.x/jetty-util/src/test/java/org/eclipse/jetty/util/log/StdErrLogTest.java